### ISSUE TITLE
Single Image "filepond--item" preview for multiple files.

### ISSUE THREAD

**Issue by elaidaomar** [pqina/filepond#745]
First of all, amazing work pqina and Rik Schennink for this amazing plugin. Love it!

Is there a way to remove the little preview ("filepond--item") for multiple images in FilePond please?

Something similar to the following example (but without the .txt obvs xD) :
![image](https://user-images.githubusercontent.com/10351224/127324921-2f4dc0ce-fd15-4251-8f0c-bf48246a86ad.png)

Say for example I need to upload a folder which has 150 attachments. Currently that would create 150 "filepond--item" which would create a lot of clutter on the page.

Once again I thank you for your work!

**Comment by rikschennink** [comment]
Do you mean grouping them under one item?

**Comment by elaidaomar** [comment]
Yes and like for example have: 15 Attachments Uploaded. Instead of one for each item.
